src/platform/unittest/TlsTest.cpp

```diff
@@ -770,6 +826,7 @@ const CXPLAT_TLS_CALLBACKS TlsTest::TlsContext::TlsClientCallbacks = {
 
 QUIC_CREDENTIAL_CONFIG* TlsTest::SelfSignedCertParams = nullptr;
 QUIC_CREDENTIAL_CONFIG* TlsTest::ClientCertParams = nullptr;
+QUIC_CREDENTIAL_CONFIG* TlsTest::CertParams = nullptr;
```

Comment:
Can you make the variable a bit more descriptive please? Also it seems everything getting allocated for it ends up being leaked, afaict.

Reply:
That is hard one. Renamed to CertParamsFromFile. While this has self-signed certificate as SelfSignedCertParams, this may have more in future. If you have other suggestions please let me know.
Added more cleanup to TearDownTestSuite